### ISSUE TITLE
Breaks with newline after comma in keyframes

### ISSUE THREAD

**Issue by jaydenseric** [thysultan/stylis.js#103]
Originally reported in styled-jsx: https://github.com/zeit/styled-jsx/issues/432#issuecomment-381144393.

This works:

```css
@keyframes test {
  0%, 50% {
    opacity: 0;
  }
}
```

And this doesn't:

```css
@keyframes test {
  0%,
  50% {
    opacity: 0;
  }
}
```

Which is unfortunate, since that's how Prettier formats CSS.

A strange `\u0` character intermittently appears after the comma, causing the keyframes to be discarded by the browser:

<img width="120" alt="screen shot 2018-04-13 at 3 02 24 pm" src="https://user-images.githubusercontent.com/1754873/38739324-ee4a329e-3f77-11e8-897f-0102358e793d.png">

**Comment by jaydenseric** [comment]
I can't get anything to blow up using https://stylis.js.org, which presumably uses the latest version of stylis (v3.5.0). I am exeriencing the issue using `styled-jsx`, which has locked the `stylis` dependency to [v3.4.10](https://github.com/zeit/styled-jsx/blob/v2.2.6/package.json#L66).

**Comment by thysultan** [comment]
It probably only manifests with the cascade option set to false, of which [https://stylis.js.org](https://stylis.js.org) does not use.

**Comment by thysultan** [comment]
In which case this might also be related to https://github.com/thysultan/stylis.js/issues/104

**Comment by thysultan** [comment]
Fixed in 3.5.1
